spec/deaf_grandma_spec.rb

```diff
@@ -9,10 +9,27 @@
   end
   
   it "says 'NOT SINCE 1964!' when we yell" do
-    #implement your test here
+    expect(script.speak("HI GRANDMA")).to eq "NOT SINCE 1964!"
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
-    #implement your test here
+    2.times { script.speak("BYE") }
```

Comment:
Nice.  We often use `before` blocks for this sort of setup.  [see here](https://relishapp.com/rspec/rspec-core/v/2-2/docs/hooks/before-and-after-hooks)